tests/unit/app/test_api.py

```diff
@@ -18,7 +18,9 @@ def test_read_root(anon_client):
     assert prev_resp.headers["location"] == "./docs"
 
 
-def test_request_summary_is_logged(caplog):
+def test_request_summary_is_logged(
+    caplog,
```

Comment:
nit: trailing comma?

Reply:
I believe the linter added this upon adjusting the file.